CONTRIBUTING.md

```diff
@@ -39,14 +28,6 @@ Every issue in this respository and on the project board should be appropriately
 
 We also have labels for each discipline and for research and project management related tasks. While this repository and project board track development work, we try to document all work related to the project here as well. 
 
-## Pull request etiquette
-
-- The submitter is in charge of merging their PRs unless the approver is given explicit permission.
-- Do not commit to another person's branch unless given explicit permission.
-- Keep pull requests as small as possible. This makes them easier to review and track changes.
-- Bias towards approving i.e. "good to merge once X is fixed" rather than blocking until X is fixed, requiring an additional review.
-- Write descriptive pull requests. This is not only something that makes it easier to review, but is a great source of documentation. 
-
 ## Branch Naming
 
-Our branch naming convention is `name/topic-or-feature`, for example: `lmm/add-contributing-doc`. 
+Our branch naming convention is `name/issue_no-description`, for example: `lmm/0000-add-contributing-doc`. 
```

Comment:
```suggestion
Our branch naming convention is `name/0000-description`, for example: `lmm/1234-add-contributing-doc`. 
```

Reply:
pushed a change for this - thank you for the suggestion!